en/telemetry/esp32_wifi_module.md

```diff
@@ -1,169 +1,159 @@
+_DroneBridge for ESP32_ can run on almost all ESP32 development boards.
+Boards and modules with an external antenna connector are recommended since these will offer more range.
 
 :::warning
-Many ESP32 modules support 3.3V and 5V power supply input, while some flight controllers (e.g. Pixhawk 4) output at 5V.
+Some ESP32 modules support 3.3V power supply input, while many flight controllers output at 5V (e.g. Pixhawk 4).
 You will need to check compatibility and step down the voltage if needed.
 :::
 
-Modules and DevKits that accept 3.3V or 5V power supply:
+::: tip
+The _DroneBridge for ESP32_ project recommends using [Officially Supported and Tested Boards](https://dronebridge.github.io/ESP32/).
+These are small, low in price, have Pixhawk-standard port for connecting to Pixhawk flight controllers, and come with _DroneBridge for ESP32_ firmware pre-installed.
+:::
+
+Other ESP32 Boards that have been tested with earlier versions of _DroneBridge for ESP32_:
 
 - [AZ-Delivery — ESP-32 DevKit C](https://www.az-delivery.de/en/products/esp-32-dev-kit-c-v4)
 - [TinyPICO — ESP32 Development Board - V2](https://www.adafruit.com/product/4335)
 - [Adafruit HUZZAH32 — ESP32 Feather Board](https://www.adafruit.com/product/3405)
 - [Adafruit AirLift — ESP32 WiFi Co-Processor Breakout Board](https://www.adafruit.com/product/4201) (requires FTDI adapter for flashing firmware)
 - [Adafruit — HUZZAH32](https://www.adafruit.com/product/4172) (requires FTDI adapter for flashing firmware)
 
-## Downloading and Flashing the Firmware
-
-[Download the firmware from the GitHub repository](https://github.com/DroneBridge/ESP32/releases) and then [follow these flashing instructions](https://github.com/DroneBridge/ESP32#installationflashing-using-precompiled-binaries).
-
-:::tip
-The instructions on Github are recommended because they are always up to date.
-Note that the parameters may differ between releases of _DroneBridge for ESP32_.
-:::
+## Installing/Updating the Firmware
 
-The main steps are:
+An easy-to-use online flashing tool is available on the official website.
+Just connect your ESP32 and click flash!
 
-1. [Download the pre-compiled firmware binaries](https://github.com/DroneBridge/ESP32/releases)
-1. Connect your DEVKit to your computer via USB/Serial bridge (most DevKits already offer a USB port for flashing and debugging)
-1. Erase the flash and flash the DroneBridge for ESP32 firmware onto your ESP32
-   - Using [Espressif Flash Download Tool](https://www.espressif.com/en/support/download/other-tools) (Windows only)
-   - Using esp-idf/esptool (all platforms)
-1. Power Cycle the ESP32
-1. [Connect to the "DroneBridge for ESP32" WiFi network and configure the firmware for your application](#configuring-dronebridge-for-esp32)
+[For more detailed information please visit the official Wiki!](https://dronebridge.gitbook.io/docs/dronebridge-for-esp32/untitled)
```

Comment:
Please change to direct link to the flashing instructions.

Reply:
Added